RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Cake Pull Request Code Analysis Addin Release Process
+
+This document describes the necessary steps to release a new version of the Cake Pull Request Code Analysis Addin.
+
+* Create a release branch (eg. `release\1.2.3`).
+* Make sure that a GitHub milestone exists for this release.
+* Make sure there were issues for all changes with the appropriate labels and the correct milestone set.
+* Make sure to have Git Release Manager installed locally `choco install gitreleasemanager.portable`.
+* Run Git Release Manager locally to create a GitHub release draft:
```

Comment:
@pascalberger the environment variables are already loaded into AppVeyor.  I was referring to you adding the same environment variables locally on your machine, so that you can run `build -target releasenotes` locally.